spec/src/main/asciidoc/microprofile-lra-spec.adoc

```diff
@@ -684,6 +684,10 @@ or may in the process of compensation (completion) the exception in these method
 result into failure condition (in JAX-RS this condition is represented by 
 500 return HTTP status code) which individual interpretation is left further unspecified.
 
+The non-JAX-RS participant methods MUST be invoked by the implementation of the 
```

Comment:
The business author can call whichever methods he wants and if that results in something undesirable then he should re-code his app - ie I don't think this is doing anything to enhance the spec.